index.html

```diff
@@ -0,0 +1,322 @@
+                <hr class="lang-rectnalge">
+                <small class="langtxt skltxt">Language</small>
+            </div>
+            <ul class="langtags ullng">
+                <li class="lng lngj"><small class="lngtxt">Javascript</small></li>
+                <li class="lng lngr"><small class="lngtxt">Ruby</small></li>
+                <li class="lng lngh"><small class="lngtxt">Html</small></li>
+                <li class="lng lngc"><small class="lngtxt">Css</small></li>
+            </ul>
+        </article>
+        <article class="frameworks">
+            <div class="frmwrk-contain">
+                <hr class="frmwrk-rectnalge">
+                <small class="skltxt frmktxt">Frameworks</small>
+            </div>
+            <ul class="langtags ulfrmwrk">
+                <li class="lng lngj"><small class="lngtxt">React.js</small></li>
+                <li class="lng lngr"><small class="lngtxt">Ruby on Rails</small></li>
+                <li class="lng lngh"><small class="lngtxt">RSpec</small></li>
+                <li class="lng lngc"><small class="lngtxt">CapyBara</small></li>
+            </ul>
+        </article>
+        <article class="skills-list">
+            <div class="skills-list-contain">
+                <hr class="skills-circle">
+                <small class="skltxt txtskl">Skills</small>
+            </div>
+            <ul class="langtags ulskill">
+                <li class="lng lngj"><small class="lngtxt">DatabaseManagement</small></li>
+                <li class="lng lngr"><small class="lngtxt">Version Control</small></li>
+                <li class="lng lngh"><small class="lngtxt">CLI</small></li>
+                <li class="lng lngc"><small class="lngtxt">API Design</small></li>
+                <li class="lng lngc"><small class="lngtxt">Web Development</small></li>
+            </ul>
+        </article>
+    </section>
+</section>
+<section class="add-contact-text">
+    <div class="text-2">
+            <p class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    </div>
+    <form class="add-info" action="https://formspree.io/f/xwkjovpk"
+    method="POST">
+        <div class="full-name">
+            <input class="border-none-1" type="text" name="name" maxlength="30" placeholder="Full name" required>
+            <hr class="container-1">
+        </div>
+        <div class="email-address">
+            <input class="border-none-2" type="email" name="email" placeholder="Email address" required>
+            <hr class="container-2">
+        </div>
+        <div class="message-text">
+            <textarea class="border-none-3" name="message" rows="8" cols="39" maxlength="500" placeholder="Enter text here" required></textarea>
+        </div>
+
+        <button class="form-button" type="submit" button-interaction="hover">         
+            <h3 class="form-button-text">Get in touch</h3>
+        </button>
+    </form>
+</section>
```

Comment:
- [ ] As suggested by the previous reviewers, the contact form section(Mandatory) should be responsive according to the requirements. The problem in your design is that you are giving exact widths to your containers(section and div), we actually avoid giving exact widths to our containers because exact widths make it impossible to achieve the responsive design. Also kindly consider avoiding giving the heights to your containers as well because we try to achieve heights by giving our elements margins and paddings.
Kindly consider making your design responsive by applying above-mentioned guidelines. you can take help from [this link](https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-natural-responsiveness), try exploring the whole section of  `Responsive Design` on this website.
your design
![qw](https://user-images.githubusercontent.com/96683843/228056946-c1206ac7-04a7-49fb-ac95-8b34130663c9.jpg)